factory-package-news/factory-package-news.py

```diff
@@ -27,6 +29,17 @@ def utf8str(content):
     return str(content, 'utf-8') if isinstance(content, bytes) else content
 
 
+# Older versions wrote uncompressed files, support both for now.
+def openZstdOrPlain(path: str):
```

Comment:
Not that I really care but I guess the coding style used here would be:

```suggestion
def open_zstd_or_plain(path: str):
```

Reply:
Done